### PR TITLE
This is my second pull request

### PR DESCRIPTION
**Pull request by jonathanmunamire**
I've already Committed the Added Pages

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -96,3 +96,194 @@ main {
...
+  border-radius: 10px;
+}
+
+.btn {
+  padding: 1em;
+  border: 1px solid #396df2;
+  border-radius: 10px;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
```
Comment (Ugboaja-Uchechi) [inline style.css:227]: - [ ] Great job with your styling 👍 . Kindly make sure you give all `buttons` interactions as seen in the Figma design 🙏 

image

File: style.css [hunk 2]
```diff
@@ -96,3 +96,194 @@ main {
...
+}
+
+.resume {
+  padding-left: 6%;
+}
+
+.downmenus {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
```
Comment (Ugboaja-Uchechi) [inline style.css]: - [ ] Kindly make sure your `about section` has a `border-top-right-radius` as seen in the `Figma` design 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @jonathanmunamire,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ugboaja-Uchechi — APPROVED** [review]
Hi @jonathanmunamire , 

## Status: APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 
- Great job with your design and functionalities 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jonathanmunamire** [comment]
Thanks so much @Ugboaja-Uchechi

**Pull request merged**